.github/workflows/release-production.yml

```
@@ -70,3 +74,64 @@ jobs:
         with:
           target_dir: charts
           token: ${{ secrets.GITHUB_TOKEN }}
+
+  install:
+    needs: chart
+    if: !contains(github.ref, '-')
+    runs-on: ubuntu-latest
+    env:
+      MPDEV: "/home/runner/bin/mpdev"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Authenticate to Google Cloud
+        id: google-auth
+        uses: google-github-actions/auth@v1
+        with:
+          workload_identity_provider: 'projects/798248771094/locations/global/workloadIdentityPools/default-pool/providers/gh-provider'
+          service_account: 'container-publisher@mirror-node-public.iam.gserviceaccount.com'
+
+      - name: Set GCloud CLI
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Setup Docker Buildx QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Authorize Docker
+        run: gcloud auth configure-docker
+
+      - name: Install k3d
+        run: curl --retry 3 -fsL https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
+
+      - name: Create k3d cluster
+        run: k3d cluster create mirror --agents 1 --timeout 5m --wait --image rancher/k3s:v1.24.3-k3s1
+        timeout-minutes: 3
+
+      - name: Get tag
+        run: echo "TAG=${GITHUB_REF#refs/*/v}" >> $GITHUB_ENV
+
+      - name: Install kubectl
+        run:  curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl"
+
+      - name: Create bin
+        run:  cd ${HOME} && mkdir "bin" && export PATH=${HOME}/bin:$PATH && echo "Path is ${PATH}"
+
+      - name: Install mpdev
+        run:  docker run gcr.io/cloud-marketplace-tools/k8s/dev cat /scripts/dev > ${MPDEV}
+          && chmod +x ${MPDEV}
+
+      - name: Install application
+        run: kubectl apply -f "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml"
+
+      - name: Build images
+        run: ./release.sh "${TAG}"
+        working-directory: charts/marketplace/gcp
+
+      - name: Install marketplace app
+        run: ${MPDEV} verify --deployer=gcr.io/mirror-node-public/hedera-mirror-node/deployer:${TAG} --parameters='{"rest.service.type":"ClusterIP", "grpc.service.type":"ClusterIP"}'
```

You should be able to omit the parameters since you set them as default in schema-test.yml, right?
```suggestion
        run: ${MPDEV} verify --deployer=gcr.io/mirror-node-public/hedera-mirror-node/deployer:${TAG}
```